src/Winium.Cruciatus/Application.cs

```diff
@@ -109,9 +122,37 @@ public void Start(string arguments)
                                Arguments = arguments
                            };
 
-            this.process = Process.Start(info);
+            this.process = Process.Start(info);           
         }
 
+        /// <summary>
+        /// Update process property by process name
+        /// </summary>
+        /// <param name="launchedAppProcessName">Launched application process name</param>
+        /// <returns>true if launched process is successfully update. false if there is error occurs</returns>
+        public void UpdateRunApplicationProcessBy(string launchedAppProcessName)
+        {
+            if (launchedAppProcessName == String.Empty)
+            {
+                return;
+            }
+
+            var processList = Process.GetProcessesByName(launchedAppProcessName);
+            if (processList.Length >= 1)
+            {
+                // Get running process base on StartTime          
+                var processSortedList = processList.OrderBy(p => p.StartTime);
+                if ((launchedAppProcessName.ToLower().Equals("iexplore")) || (launchedAppProcessName.ToLower().Equals("firefox")) || (launchedAppProcessName.ToLower().Equals("chrome")))
```

Comment:
This is like hardcoded, any in future we may have other similar apps like this to append?

Any general solution for this?